### ISSUE TITLE
Some items present the References in the Attachments panel and some don't

### ISSUE THREAD

**Issue by nukedone** [Zettlr/Zettlr#1251]
I have a very strange situation that when clicking randomly on my files (inside one directory), for some files references pop pu immediately, and for some (the most recent file that I created) the references don't refresh. They are visible when I export the document both in PDF and LibreOffice, but something strange is happening with the Attachments Panel.

**Comment by bytersproblem** [comment]
I am using the latest 1.7.5 version on Xubuntu 20.04

**Comment by bytersproblem** [comment]
I kinda resolved my own issue (if i wasn't the issue :)). In one reference i put [idem; also benki2001]. It seems that "idem" couldn't be parsed. 
Now I am OK. 
Greeting for the great program.

**Comment by the-solipsist** [comment]
I've faced a similar/the same problem even without the *idem* or any other unrecognizable reference. I would have added a new reference via Zotero. Zotero auto-exports a `.bib` file thanks to BetterBibTex. I confirm that the new reference is there in the file by opening it and checking.  I try multiple times to add the new reference in Zotero by pressing `@` and searching for the new reference in the auto-complete suggestions, and it isn't there. For me, closing and re-opening Zettlr fixes the problem, as that forces Zettr to re-read the `.bib` file.  The other option, which is what I end up doing, is to copy the citekey from Zotero (Ctrl+Shift+C in Linux), and paste it into Zettlr.

**Comment by bytersproblem** [comment]
With betterbibtex I export to Better CLS Json (I think that I got it right). 
No problem now. Maybe you should try. WIth the option to record updates.

**Comment by nathanlesage** [comment]
Yes, each part of a block citation **must** have a cite key. This is a somewhat undesirable behavior in the City library. Transferring issue …
